examples/oci/config.yaml

```diff
@@ -1,7 +1,11 @@
 oci:
   default:
     compartment_ocid: ocid1.compartment.oc1..aaaaaaaahr7aicqtodxmcfor6pbqn3hvsngpftozyxzqw36gj4kh3w3kkj4q
-    vcn_ocid: ocid1.subnet.oc1.ap-seoul-1.aaaaaaaa5c6wndifsij6yfyfehmi3tazn6mvhhiewqmajzcrlryurnl7nuja
+    #image_tag_general: skypilot:oci-ubuntu-20_04
+    #image_tag_gpu: skypilot:oci-ubuntu-NVIDIA-VMI-20_04
 
-  #mycluster30:
-    #vcn_ocid: ocid1.subnet.oc1.ap-seoul-1.aaaaaaaa5c6wndifsij6yfyfehmi3tazn6mvhhiewqmajzcrlryurnl7nbbb
+  ap-seoul-1:
+    vcn_subnet: ocid1.subnet.oc1.ap-seoul-1.aaaaaaaa5c6wndifsij6yfyfehmi3tazn6mvhhiewqmajzcrlryurnl7nuja
+
+  us-ashburn-1:
+    vcn_subnet: ocid1.subnet.oc1.iad.aaaaaaaafbj7i3aqc4ofjaapa5edakde6g4ea2yaslcsay32cthp7qo55pxa
```

Comment:
We may want to find the vcn with name `skypilot-vcn` when the user does not specify it in the config.yaml

If the user want to specify the `vcn` in the config.yaml, is it possible that the user only provide the name of the `vcn_subnet` that applies to all the regions, and skypilot will try to find the correct id when we go to that region?

This is to be aligned with the design we had for the AWS VPC, where the user can specify a `vpc_name` in the `config.yaml`, and SkyPilot will automatically try to find the id to fill in the VM provisioning configuration. Reference: https://github.com/skypilot-org/skypilot/pull/1512
cc'ing @concretevitamin

Reply:
Yes, like @Michaelvll said, the intention of the AWS name-based VPC design is that a user can conveniently specify the same name and have it work for many regions/zones, even though the name may map to different VPC IDs in different regions.

I think in the OCI case, if the OCI user that @HysunHe is working with strongly prefers specifying the specific VCN subnet IDs for each region, that could work too. @HysunHe: can you confirm which approach the user prefers?